### PR TITLE
Add keyauth.win

### PR DESCRIPTION
**Pull request by wnelson03**
Add icon

### CODE REVIEW

File: entries/r/restorecord.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "RestoreCord": {
+        "domain": "restorecord.com",
+        "url": "https://restorecord.com/login/",
```
Comment (hkamran80) [inline entries/r/restorecord.com.json]:
```suggestion
```

You don't need `url` if going to the base domain (e.g. `https://restorecord.com`) works.

File: entries/r/restorecord.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        "tfa": [
+            "totp"
+        ],
+        "categories": [
+            "backup",
+			"communication"
```
Comment (hkamran80) [inline entries/r/restorecord.com.json]:
```suggestion
            "backup"
```

I would classify it as just `backup`

File: img/r/restorecord.com.svg [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+-102 89 -102 13 -2 280 614 293 676 10 43 -7 70 -52 83 -51 14 -745 13 -812
+-1 -61 -13 -107 -50 -137 -111 -12 -23 -78 -176 -148 -337 -88 -207 -125 -304
+-123 -325 5 -52 38 -61 233 -61 l170 0 9 43 c5 23 12 42 15 42 4 0 49 -43 101
+-95 l94 -94 -100 -47 c-54 -25 -119 -55 -143 -65 l-43 -19 6 22 c3 13 9 33 12
+46 l6 22 -196 0 c-210 0 -244 6 -294 53 -38 35 -52 72 -52 132 0 51 14 91 126
+353 185 435 211 480 308 537 87 51 116 54 611 54 288 1 474 -3 497 -9z"/>
+<path d="M1178 1463 c-13 -16 -48 -94 -48 -108 0 -10 272 -5 292 6 22 12 62
+94 53 109 -4 6 -64 10 -144 10 -119 0 -140 -2 -153 -17z"/>
+</g>
+</svg>
```
Comment (hkamran80) [inline img/r/restorecord.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 256 256"><path d="M35.3 2.1C20.3 5 7.3 16.6 3 30.9c-2 6.4-2 9-1.8 99.6l.3 93 3.1 6.6a44.6 44.6 0 0 0 25.2 23.2c5.7 2.2 6.2 2.2 92.2 2.5 56.4.2 89.3-.1 94.5-.7a42.2 42.2 0 0 0 36.9-31.6c1.4-5.1 1.6-17.7 1.6-95.5 0-72.6-.3-90.5-1.4-94.6A43.5 43.5 0 0 0 220 2C213.6.8 41.5.8 35.3 2.1zM199.2 68c9.2 2.6 13 9.8 10.4 19.8-1.9 7.3-31.6 76.7-35.1 82.1-5.1 7.6-16.7 13.3-20.2 9.8-.8-.8-9-10.6-18.2-21.8l-16.7-20.4-5.8-.3c-7-.4-8.3.6-11.1 7.8-3.3 8.6-5.3 10.3-12.3 10.8-8 .6-11.2-.3-11.2-2.9 0-3.7 23.1-57.9 25.4-59.4 3.2-2.3 64.8-2.2 67 .1 2.6 2.5 1.9 6.5-3.5 19.3-5.7 13.4-9.4 17.8-19.1 22.4l-4.9 2.3 8.3 10.1c4.6 5.6 8.6 10.2 8.9 10.2 1.3.2 28-61.4 29.3-67.6 1-4.3-.7-7-5.2-8.3-5.1-1.4-74.5-1.3-81.2.1-6.1 1.3-10.7 5-13.7 11.1-1.2 2.3-7.8 17.6-14.8 33.7-8.8 20.7-12.5 30.4-12.3 32.5.5 5.2 3.8 6.1 23.3 6.1h17l.9-4.3c.5-2.3 1.2-4.2 1.5-4.2.4 0 4.9 4.3 10.1 9.5l9.4 9.4-10 4.7-14.3 6.5-4.3 1.9.6-2.2 1.2-4.6.6-2.2H79.6c-21 0-24.4-.6-29.4-5.3-3.8-3.5-5.2-7.2-5.2-13.2 0-5.1 1.4-9.1 12.6-35.3 18.5-43.5 21.1-48 30.8-53.7 8.7-5.1 11.6-5.4 61.1-5.4a587 587 0 0 1 49.7.9z"/><path d="M117.8 109.7a42.2 42.2 0 0 0-4.8 10.8c0 1 27.2.5 29.2-.6 2.2-1.2 6.2-9.4 5.3-10.9-.4-.6-6.4-1-14.4-1-11.9 0-14 .2-15.3 1.7z"/></svg>
```

Here's a minified SVG

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Can you provide a documentation URL or screenshot showing proof of 2FA?

**Comment by wnelson03** [comment]
@hkamran80 they don't have documentation on the 2FA currently, I'll suggest they add it though.

here's a screenshot. I deleted out parts of the QR code and secret plus it's a brand new account I made that I don't have anything important on.

image

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**